### PR TITLE
Add Netgear ReadyNAS and ReadyData appliances

### PR DESCRIPTION
**Pull request by reynico**

### CODE REVIEW

**Comment by SuperQ** [comment]
This needs a rebase.

**Review by SuperQ — CHANGES_REQUESTED** [review]
Rebase required.

File: generator/generator.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ modules:
       ifType:
         type: EnumAsInfo
 
+  readynas:
+    walk:
+      - 1.3.6.1.2.1.1.5               # Name
+      - 1.3.6.1.2.1.25.1.1            # Uptime
+      - 1.3.6.1.4.1.4526              # Raid/Disks status
+      - 1.3.6.1.2.1.25.3.3.1.2        # CPU status
```
Comment (SuperQ) [inline generator/generator.yml:36]: With the refactoring of the generator, we only put device-specific walks in modules. The other walks are accounted for in other modules. For example the `hrDevice` module.


Also, let's be a little more specific in our walk specifications. Not the whole enterprise.

```suggestion
      - 1.3.6.1.4.1.4526              # Raid/Disks status
```

**Comment by reynico** [comment]
up to this point, and given the divergence between branches, I think it'd better start from scratch (-: @SuperQ what do you think?

**Pull request closed without merging**